### ISSUE TITLE
Canvas editing in SDK 22/04 seems not to work

### ISSUE THREAD

**Issue by StrandedAlien** [armory3d/armory#2478]
Thank you for contributing to Armory!

**Description**
Editing button for canvas does not seem to work anymore

**To Reproduce**
Use Default cube scene, new canvas, edit canvas throws following error in console

X Error: BadRROutput (invalid Output parameter)

Push edit canvas again generates more identical errors like this

**Test File**
Default cube

**Comment by MoritzBrueckner** [comment]
Hi, do you use Linux? It seems like this issue has happened before, see https://forums.armory3d.org/t/linux-krom-doesnt-compile-in-sdk-2010/4252 and https://github.com/armory3d/armory/issues/1909 for details (it seemed to happen when the primary display wasn't properly set up). It was apparently fixed by https://github.com/Kode/Kinc/pull/563, but maybe there's a regression. I think that the Kinc window handling on Linux got major updates recently, maybe those updates have re-introduced this issue?

**Comment by StrandedAlien** [comment]
Yep, Manjaro/Arch Linux. Its a rolling distro with plenty of updates. Compiling to Linux-HL is broken, i know in 22/04. But i am reffering to a Canvas which does not open when i create a new one and push Edit Canvas (Under Add trait- UI). The button does not seem to do anything anymore and prevents me entirely from using Zui menus. Hope this made it clearer now.
![Canvas_not_opening](https://user-images.githubusercontent.com/89656885/162638157-ca46a56e-05ff-4201-8dca-ed7c51dab03b.jpg)

**Comment by tong** [comment]
Same issue here if i my primary display is disconnected.
When it's connected the window opens but the application fails with:
```sh
Trace: TypeError: Cannot read property 'get_frequency' of null
    at Function.kha_Scheduler.start (<anonymous>:6063:36)
    at Function.kha_SystemImpl.init (<anonymous>:7048:16)
    at Function.kha_System.start (<anonymous>:6697:17)
    at Function.Main.main (<anonymous>:92:13)
    at <anonymous>:32856:6
    at <anonymous>:32857:3
```

**Comment by StrandedAlien** [comment]
Does not really appear as the same error to me, as i only get this one error line
X Error: BadRROutput (invalid Output parameter)
every time i click on edit canvas.
But no canvas window opens.

**Comment by tong** [comment]
If i build armorcore myself i get an empty window and:
```sh
Display 0 not connected.
Display 0 not connected.
Display 0 not connected.
Display 0 not connected.
Using OpenGL version 4.6 (forward-compatible).
```
… with  primary display set up and connected (check with: `xrandr --current`).

**Comment by StrandedAlien** [comment]
Latest working version with unchanged hard-and software setup is 22/02. Not working in 22/03 and 22/04. 
![Canvas22-02](https://user-images.githubusercontent.com/89656885/162842066-ec98baf1-afd8-4bf8-985c-ddfcb850590c.jpg)
